website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -1842,6 +1847,9 @@ transform's intermediate data changes type multiple times.
 {{< github_sample "/apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py" pipeline_monitoring_composite >}}
 {{< /highlight >}}
 
+Note that because `Count` is itself a composite transform,
```

Comment:
I'd either format this as a note (see other notes on this page) or remove "Note that"

Reply:
done